src/util.js

```diff
@@ -9,6 +9,17 @@ export function assign(obj, props) {
 	return obj;
 }
 
+export function getMatchingRoutes(routes, currentRoute) {
```

Comment:
Was thinking this could be trimmed to:

```js
export function getMatchingRoutes(routes, currentRoute) {
	let out = [], route, matches, i, sorted = routes.slice().sort(pathRankSort);
	for (i=0; i<sorted.length; i++) {
		route = sorted[i];
		matches = route.attributes && exec(currentRoute, route.attributes.path, route.attributes);
		if (matches) out.push({ route, matches });
	}
	return out;
}
```

Reply:
After rebase, current size of the lib on your repository:

```
$ npm run size

> preact-router@2.5.7 size ~/Desktop/github/preact-router
> size=$(gzip-size $npm_package_main) && echo "gzip size: $size / $(pretty-bytes $size)"

gzip size: 1991 / 1.99 kB
```

With the suggested refactoring it changes to **1995 / 2 kB**. So it increase the size.
On the basis of your previous inputs I'm going to ignore the suggested refactoring; is it OK?